data/brands/shop/supermarket.json

```diff
@@ -9085,21 +9149,54 @@
         "brand": "מגה",
         "brand:en": "Mega",
         "brand:he": "מגה",
+        "brand:wikidata": "Q12403907",
         "name": "מגה",
         "name:en": "Mega",
         "name:he": "מגה",
+        "name:ru": "Мега",
```

Comment:
I'm curious that you provided Russian names more than once, as far as I know Russian is not spoken in Israel, can you explain why?

Reply:
Hi, it is spoken and used a lot in OSM data, also about 15-25% of Israel is Russian-speaking (mostly Jews from Russia, Ukraine, Belorussia and other ex-USSR countries)
https://en.wikipedia.org/wiki/Russian_language_in_Israel

Reply:
can I resolve it..? please re-open if needed